opentargets.json

```diff
@@ -203,6 +197,9 @@
         },
         "diseaseFromSourceMappedId": {
           "$ref": "#/definitions/diseaseFromSourceMappedId"
+        },
+        "pmcIds": {
+          "$ref": "#/definitions/pmcIds"
         }
       },
       "required": [
```

Comment:
`targetFromSource` needs to be removed from the required fields.

Reply:
Oh, yes, that's true.